gcc/rust/parse/rust-parse-impl.h

```diff
@@ -533,7 +538,10 @@ Parser<ManagedTokenSource>::parse_inner_attribute ()
   if (!skip_token (LEFT_SQUARE))
     return AST::Attribute::create_empty ();
 
-  AST::Attribute actual_attribute = parse_attribute_body ();
+  auto values = parse_attribute_body ();
+  auto actual_attribute = AST::Attribute (std::move (std::get<0> (values)),
+					  std::move (std::get<1> (values)),
+					  std::get<2> (values), true);
```

Comment:
same here. And when we ~~rewrite it in Rust~~ switch to C++14/17, we can use `auto [path, input, loc] = parse_attribute_body ();`

Reply:
Fixed.